### DISCUSSION TITLE
NXDomain after unpausing a project

### DISCUSSION THREAD

**Discussion by ar1a** [supabase/supabase#2843] [Questions] [answered]
Hey, just went to resume work on a project and unpaused it (https://uxtxhzqpveqocjohidmf.supabase.co) but the domain hasn't updated, just getting NXDomain

do i just need to wait it out?

**Comment by dragarcia** [comment, marked as answer]
Hi @ar1a 👋 

We're taking a look at this now. Will update you as we determine the exact problem.

**Reply by dragarcia** [reply]
Hi @ar1a!

We've taken a look under the hood and resolved the problem at hand. Would you mind checking your project once more?

**Reply by ar1a** [reply]
yep, all works from my end, thanks!

**Reply by ar1a** [reply]
I just tried authentication at https://uxtxhzqpveqocjohidmf.supabase.co/auth/v1/authorize?provider=google and i'm running into this returned

```
{
  "message":"An invalid response was received from the upstream server"
}
```

is this from the same failure to unpause issue, or is it on my end? @dragarcia

**Reply by ar1a** [reply]
same with trying to send a magic link in the dashboard, i just get 


![image](https://user-images.githubusercontent.com/8436007/129727588-a4f4d3b8-7841-4d38-bd08-9c0e01ca6909.png)
